templates/ui-ingress.yaml

```
@@ -0,0 +1,31 @@
+{{- if (and (ne (.Values.ui.ingress.enabled | toString) "-") .Values.ui.ingress.enabled) }}
+{{- $serviceName := printf "%s-%s" (include "consul.fullname" .) "ui" -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "consul.fullname" . }}-ingress
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  {{- if .Values.ui.ingress.annotations }}
+  annotations:
+    {{ tpl (toYaml .Values.ui.ingress.annotations) . | nindent 4 | trim }}
```

This should be without toYaml to be inline with the rest of the code base.
```suggestion
    {{ tpl .Values.ui.ingress.annotations . | nindent 4 | trim }}
```

---

Hi, I've remove `toYaml` function and updated code already, please help review again.